src/cai_lab_to_nwb/zaki_2024/interfaces/zaki_2024_edf_interface.py

```diff
@@ -1,33 +1,29 @@
-from pydantic import FilePath
 from pathlib import Path
-
 from neuroconv.basedatainterface import BaseDataInterface
 from pynwb import NWBFile, TimeSeries
 from pynwb.device import Device
-
 from mne.io import read_raw_edf
+from datetime import datetime, timedelta
+import numpy as np
 
 
 class Zaki2024EDFInterface(BaseDataInterface):
-
-    def __init__(self, file_path: FilePath, verbose: bool = False):
-
+    def __init__(
```

Comment:
I think these should be conversion options and not init arguments, as you might want to use run-time information (from other interfaces for example) to determine them.